contracts/eth/nearprover/hardhat.config.js

```diff
@@ -13,5 +35,23 @@ module.exports = {
                 runs: 1000
             }
         }
+    },
+    networks: {
+        ropsten: {
+            url: `https://ropsten.infura.io/v3/${INFURA_API_KEY}`,
+            accounts: [`0x${ETH_PRIVATE_KEY}`],
```

Comment:
Please add an explicit note in the readme that the `ETH_PRIVATE_KEY` should be set without '0x'

Reply:
Fixed in f5183d1